frontend/src/v5/ui/components/shared/revisionDetails/components/revisionsListHeaderLabel/revisionsListHeaderLabel.component.tsx

```diff
@@ -21,8 +21,12 @@ import { Container } from './revisionsListHeaderLabel.styles';
 
 type IDashboardListHeaderLabel = {
 	width?: number;
+	tabletWidth?: number;
+	mobileWidth?: number;
 	className?: string;
 	children?: ReactNode;
+	hideBelowTablet?: boolean;
+	marginRight?: boolean;
```

Comment:
Question: in which case would we need marginRight to be true?

Reply:
I added this extra styling via styles, using the prop for this is actually a bit redundant.